### DISCUSSION TITLE
Only render image when cursor over path?

### DISCUSSION THREAD

**Discussion by semi710** [3rd/image.nvim#14] [Q&A] [answered]
is it possible to only render image when cursor is over path of the image as this will make ueberzug one faster?

**Comment by 3rd** [comment, marked as answer]
@niksingh710 That's a great idea, I just pushed a new option for the markdown & neorg integrations in: https://github.com/3rd/image.nvim/commit/1abfecb67aeb103b0d597f1e8098995f2a72934d

This is how it looks like:

https://github.com/3rd/image.nvim/assets/59587503/5e7cfc84-3453-45b6-886e-a7d7c18b4b37

Try it out & let me know how it works, thanks for the contribution!

**Reply by semi710** [reply]
Works perfectly as expected.
just waiting for the vimwiki entry to be added if posssible that will make it a charm.........

**Reply by semi710** [reply]
@3rd add an option delay value so that if user is scrolling fast. then cursor being on the path will not try to render the image and will keep it smooth.
and if user holds on the image path for a x amount of time then it should render.
or add an manual invoke option like `shift+k` hover to preview image then if user want to see then image then only they can see it.
as few remote images are big in size and it is not always that we want to preview them.

**Reply by mehalter** [reply]
Being able to add a mapping to hover preview an image would be amazing!
